library/globals.inc.php

```diff
@@ -493,7 +493,8 @@
     'gbl_currency_symbol' => array(
       xl('Currency Designator'),
        'text',                          // data type
-       '$',                             // default
```

Comment:
ISO 4217.  Dollar (USD), Euro, Yen and BPS are four most commonly traded currencies, but providing currency code based conversions are a bit outside our scope, despite the folks doing business in the borderlands.